tests/test_data/ingoodflavor.com/ingoodflavor_2.json

```diff
@@ -0,0 +1,83 @@
+{
+  "author": "In Good Flavor",
```

Comment:
We could be slightly more precise here, by gathering  `Thao | In Good Flavor` from the author info on the page.